src/scrapers/VenuesScraper.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import querystring from 'querystring';
+import R from 'ramda';
+
+import BaseTask from './BaseTask';
+
+const NUS_API_URL = 'http://nuslivinglab.nus.edu.sg/api_dev/api/';
+const FIELDS = [];
+const MAX_INSERT_SIZE = 999 / 4;
```

Comment:
Why is this not an integer?

Reply:
"To prevent excessive memory allocations, the maximum value of a host parameter number is SQLITE_MAX_VARIABLE_NUMBER, which defaults to 999"

A float seems to work without error.

Reply:
It doesn't make any sense for this to be anything other than an integer. Should also comment on why we're changing this because this parameter has a value of 1000 by default.